### PR TITLE
Updated Vuex Section Documentation

### PR DESCRIPTION
**Pull request by capndave**
Hi I added a section to the docs on how to emit events from the Vuex store. I edited the entire section for grammar and removed/simplified some of the language that I found confusing or repetitive. Hopefully you'll find it clearer and easier to read! Please take or leave whichever parts you find helpful. I'm happy to work more on the docs or to tackle other issues if you like as well:)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,11 +210,21 @@ export default new Vuex.Store({
...
 
-#### Namespaced vuex modules
+#### Sending Events  
+
+Events can be sent to the Socket.IO server by calling `this._vm.$socket.emit` from a Vuex mutation or action. Mutation or action names are not subject to the same naming requirements as above. Event data must be sent in string format.
+
+```
+  actions: {
+    emitSocketEvent (data) {
+      this._vm.$socket.emit('eventName', JSON.stringify(data))
```
Comment (probil) [inline README.md]: The API was changed in v4:
```diff
- this._vm.$socket.emit
+ this._vm.$socket.client.emit
```

[Check release docs](https://github.com/probil/vue-socket.io-extended/releases/tag/v4.0.0)

**Comment by capndave** [comment]
I changed `$socket.emit` to  `$socket.client.emit`, and added syntax highlighting. Unfortunately you can't do inline code with syntax highlighting in markdown, so it doesn't read quite as nicely. I'll change it back to inline w/o syntax highlighting if you prefer, but here it is!

File: README.md [hunk 2]
```diff
@@ -209,11 +210,25 @@ export default new Vuex.Store({
 })
 ```
 
-#### Namespaced vuex modules
+#### Emitting Events  
+
+Events can be sent to the Socket.IO server by calling 
+```js
+this._vm.$socket.client.emit
```
Comment (probil) [inline README.md]: Use it like so: `this._vm.$socket.client.emit` 
no need for new lines and syntax highlight here

I'll merge after that change

File: README.md [hunk 3]
```diff
@@ -209,11 +210,25 @@ export default new Vuex.Store({
...
+```
+from a Vuex mutation or action. Mutation or action names are not subject to the same naming requirements as above. Event data must be sent in string format.
```
Comment (probil) [inline README.md]: Event data can be send in any format (not just string)
https://socket.io/docs/client-api/#socket-emit-eventName-%E2%80%A6args-ack

**Comment by probil** [comment]
@capndave merged as part of #414

File: README.md (hunk 2 above)
Comment (capndave) [reply README.md]: Awesome, thanks!

File: README.md (hunk 3 above)
Comment (capndave) [reply README.md]: Good to know, thanks!

**Pull request closed without merging**